components/style/restyle_hints.rs

```diff
@@ -341,16 +341,38 @@ struct Dependency {
 #[derive(Debug)]
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 pub struct DependencySet {
```

Comment:
Please add a comment above this struct that there are measurable performances wins from storing these separately.